src/commands/infra/generate.ts

```diff
@@ -353,13 +360,15 @@ export const generateConfigWithParentEqProjectPath = async (
     writeTfvarsFile(spkTfvarsObject, parentDirectory, SPK_TFVARS);
     await copyTfTemplate(templatePath, parentDirectory, true);
   } else {
+    // TOFIX: are these warnings? if yes, please write some comments.
     logger.warn(`Variables are not defined in the definition.yaml`);
   }
```

Comment:
Yes, this is a warning.

Reply:
Consider the case where the only common configuration is just backend configuration, and no common variable configuration. Thus, it is not "necessary" for a parent `definition.yaml` to have a variables block in a multi-cluster.

@NathanielRose do you agree?

Reply:
Agreed, we should be able to support common backend configs without enforcing common config between leaf and parent.